schema/collections.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import type { GraphQLFieldConfig } from "graphql"
+import { pageable } from "relay-cursor-paging"
+import { connectionFromArraySlice } from "graphql-relay"
+import _ from "lodash"
+import gravity from "../lib/loaders/gravity"
+import cached from "./fields/cached"
+import { artworkConnection } from "./artwork"
+import filterArtworks, { filterArtworksArgs } from "./filter_artworks"
+import { queriedForFieldsOtherThanBlacklisted, parseRelayOptions } from "../lib/helpers"
+import { GravityIDFields, NodeInterface } from "./object_identification"
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLBoolean } from "graphql"
+
+const CollectionType = new GraphQLObjectType({
+  name: "Collection",
+  interfaces: [NodeInterface],
+  isTypeOf: obj => _.has(obj, "name") && _.has(obj, "private") && _.has(obj, "default"),
+  fields: {
+    ...GravityIDFields,
+    cached,
+    artworks_connection: {
+      type: artworkConnection,
+      args: pageable({}),
+      resolve: ({ id }, options, request, { rootValue: { accessToken, userID } }) => {
+        const gravityOptions = parseRelayOptions(options)
+        gravityOptions.user_id = userID
+        gravityOptions.total_count = true
+
+        // TODO: what do to with this?
+        delete gravityOptions.page
+
+        return gravity
+          .with(accessToken, { headers: true })(`collections/artworks`, gravityOptions)
```

Comment:
Yeah, [I was hoping](https://trello.com/c/YisnjIlL/70-artsy-login-which-follows-your-instagram-and-looks-for-artworks-which-saves-them) to be able to the sets for not just saved-artworks in the future, but at we are still reflecting the gravity API this way. (I modified the line this was commented on to use the `:id` from a request.